.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Preguntar si el software está dentro del alcance de rOpenSci
-about: ¿Quieres confirmar que tu paquete encaja para revisión? Usa esta plantilla para obtener repuesta de las editoras en Español (Experimental)
+about: ¿Quieres confirmar que tu paquete encaja para revisión, o quieres envia un paquete de estadística? Usa esta plantilla para obtener repuesta de las editoras en Español (Experimental)
```

Comment:
`^1:` The "r" seems missing. I suggest:
"o quieres envia`{+r+}` un paquete de estadistica?"

`^2:` Also I mildly prefer:
"paquete `[-de estadistica-]{+estadistico+}`" 
I think it's like the difference between saying "package of statistics" versus "statistical package".